breakout_1.md

```diff
@@ -0,0 +1,182 @@
+# Breakout Room #1
+
+## Background
+You and your collaborators have made a recent discovery that will surely result in winning the prestigious
+Nobel Peach Prize, awarded to the group of biomechanists with the best peach-related figure made entirely with 
+code! Your team has been working for months and have just a few bugs to fix before submitting the figure to 
+Nature for publication. Your team will be able to quickly fix these bugs in parallel thanks to Git, a version control 
+system.
+
+The code used to generate your figure is comprised of an upper-level script `ASB_Tutorial/main_script.m`, which 
+calls five numbered subscripts (e.g. `ASB_Tutorial/subscripts/script_1.m`). The data required to generate your 
+figure is stored as CSV files at `ASB_Tutorial/data/`. 
+
+If you need help remembering the git functions, refer to `ASB_Tutorial/git_cheatsheet.md`.
+
+## Objective
+Work together to fix the bug in each subscript and generate the figure by running `main_script.m`.
+
+## 1. Organize
+Your project leader has added you as collaborators on the remote repository on Github. If this hasn't happened yet, 
+request help from the Tutorial Team.
+
+#### a. Assign Tasks
+You will divide and conquer the bugs. There is one bug in each of the five subscripts. Assign each team member a 
+subscript to fix. If there are more subscripts than team members, assign multiple subscripts for some members. 
+
+#### b. Branch Out
+Once subscripts are assigned, each team member should make their own branch of the repository. This allows team 
+members to simultaneously make changes without affecting each other or the script stored in the remote (online) 
+repository. Name each branch after the respective subscript(s) that will be fixed. 
+
+1. Navigate to the main page of the repository on www.github.com
+
+2. Click the branch selector menu:    
+![](media/branch-selection-dropdown.png)
+3. Type in a unique name for your branch (name of subscript(s) to be fixed), then select **Create branch**:    
+![](media/branch-creation-text.png)
+
+#### c. Clone repository
+If you're going to be changing code, you need a copy of it on your machine. This is done through a process
+called "cloning", where you download a local copy of a remote repository. To do this, you will use Git Bash or Terminal,
+depending on your operating system. *This tutorial will refer to Git Bash, but the functions are the same in Terminal.*
+ 
```

Comment:
This cloning should be before them splitting up?

Reply:
idk yet. If they make branches on github and then clone, branches come along. If they already clone and then need to make branches, they will have to pull those changes. That's not exactly a bad thing, but we have to think about the order of operations.

Reply:
Makes sense. The order seems a little odd to me but your method would be less steps.